t/220-mvcc.t

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env tarantool
+local qc = require('queue.compat')
+local log   = require('log')
+if not qc.check_version({2, 6, 1}) then
+    log.info('Tests skipped, tarantool version < 2.6.1')
+    return
+end
+local yaml  = require('yaml')
+local fiber = require('fiber')
+
+local test  = require('tap').test()
+test:plan(6)
+
+local queue = require('queue')
+local state = require('queue.abstract.state')
+
+local tnt   = require('t.tnt')
+tnt.cfg{memtx_use_mvcc_engine = true}
+
+local engine = 'memtx'
+
+test:ok(rawget(box, 'space'), 'box started')
+test:ok(queue, 'queue is loaded')
+
+local tube = queue.create_tube('test', 'fifo', { engine = engine })
+test:ok(tube, 'test tube created')
+test:is(tube.name, 'test', 'tube.name')
+test:is(tube.type, 'fifo', 'tube.type')
+
+test:test('concurent take', function(test)
+    test:plan(151)
+
+    local channel = fiber.channel(1000)
+    test:ok(channel, 'channel created')
+
+    local res = {}
+    for i = 1, 50 do
+        fiber.create(function(i)
+            local taken = tube:take(1)
```

Comment:
AFAIU `take()` is fixed in the next commit and should still be broken here.

Reply:
changed